src/rp2_common/hardware_uart/uart.c

```diff
@@ -75,23 +75,25 @@ uint uart_set_baudrate(uart_inst_t *uart, uint baudrate) {
     uint32_t baud_rate_div = (8 * clock_get_hz(clk_peri) / baudrate);
     uint32_t baud_ibrd = baud_rate_div >> 7;
     uint32_t baud_fbrd = ((baud_rate_div & 0x7f) + 1) / 2;
-    invalid_params_if(UART, (baud_ibrd > 65535) || (baud_ibrd == 0));
```

Comment:
I'm not sure how these macros are intended to be used, but is it "consistent" that this `invalid_params_if` has been removed, but the `invalid_params_if(UART, baudrate == 0);` is still there on line 74? :shrug:

Reply:
ha i didn't notice that

Reply:
0 is invalid because we would rather not divide by it :)